### DISCUSSION TITLE
Adblock Disabled On Launch

### DISCUSSION THREAD

**Discussion by Clarityhk2023** [qnblackcat/uYouPlus#1068] [Q&A] [answered]
Using the latest version [v18.08.1-2.3.1], recently about a week or two ago I started noticing every time the app launches the Adblock is automatically switched off in the uYou settings. Every time I force close the app and relaunch it goes back to being off again. This never happened before on previous versions, what's happening? I am experiencing the same issue on two different iPhones and my iPad as well. Installed via Sideloadly.

**Comment by qnblackcat** [comment, marked as answer]
https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-the-timestamp-of-videos-in-history-playlist-watch-later-is-missing

However, uYou 3.0 and higher fixed this issue. Thus it will not be disabled anymore

**Reply by Clarityhk2023** [reply]
Thank you! <3
